gaia/parser.py

```diff
@@ -35,12 +40,29 @@
     gaia.geo.geo_inputs, inspect.isclass) if x[0].endswith('IO')])
 
 
+def load_plugins():
```

Comment:
@mbertrand I think there might be a better place for this function. Not entirely sure if parser is the right place for it. Thoughts?